Resources/Prototypes/secret_weights.yml

```diff
@@ -1,8 +1,30 @@
 - type: weightedRandom
   id: Secret
-  weights:
+  weights: #these are the presets!!! NOT gamerules.
     Nukeops: 0.20
     Traitor: 0.50
     Zombie: 0.10
     Revolutionary: 0.20
```

Comment:
The lrp one is just a copy of this with extended added

Reply:
Nah, I stole some weight from revs so it still adds to 1, not that adding to 1 really matters other than readability.

Reply:
Its still a copy

Reply:
Good catch, I goofy.